src/tag/tag.jsx

```diff
@@ -174,6 +177,8 @@ class Tag extends Component {
             <span
                 className={`${prefix}tag-close-btn`}
                 onClick={this.handleTailClick}
+                role="button"
+                aria-label={locale.delete}
```

Comment:
We add `Tag` in our locale `src/locale/xxx.js`, but some users have their own language package, if they don't add `Tag` in their own files, `this.props.locale` will be `undefined`, `locale.delete` will cause an error.

Reply:
I saw that `Tag` was not in any `locale/xxx.js` so i added it for all of them. Is this what you mean? Or do you mean that users can inject other languages we don't currently support like Russian?